homeassistant/components/sensor/google_travel_time.py

```diff
@@ -110,12 +113,22 @@ def run_setup(event):
 
         sensor = GoogleTravelTimeSensor(
             hass, name, api_key, origin, destination, options)
+        hass.data[DATA_KEY].append(sensor)
 
         if sensor.valid_api_connection:
             add_entities_callback([sensor])
 
+    def update(service):
+        """Update service for manual updates."""
+        entity_id = service.data.get('entity_id')
+        for sensor in hass.data[DATA_KEY]:
+            if sensor.entity_id == entity_id:
+                sensor.update(no_throttle=True)
+                sensor.schedule_update_ha_state()
+
     # Wait until start event is sent to load this component.
     hass.bus.listen_once(EVENT_HOMEASSISTANT_START, run_setup)
+    hass.services.register(DOMAIN, 'google_travel_sensor_update', update)
```

Comment:
Should only register the service once, not for every sensor.

Reply:
Done :+1: